test/app/src/classes/api.js

```diff
@@ -3,7 +3,7 @@ const path = require('path');
 const express = require('express');
 const routeBuilder = require('express-routebuilder');
 
-const Endpoints = require('../../../../');
+const Endpoints = require('../../../../src');
```

Comment:
this is using the es6 so we don't have to watch and recompile on the fly, but we should probably be testing against the es5 and have a watch/rebuild going (urggh).